### PR TITLE
Change wording on Upgrade instructions to explicitly indicate upgrade path

### PR DESCRIPTION
**Pull request by vollmerk**
Adjust the wording to explicitly state that you should upgrade to each intermediate version, and run the db:migrate, previous wording lead me to believe that I could go from 1.5 -> 1.9 as long as I did the steps listed in "Preparation" on each version. The db:migrate didn't seem to be an "additional" step of previous versions, rather the final step.

### CODE REVIEW

File: _includes/manuals/1.9/3.6_upgrade.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ Please note the following known issue:
 * Hosts may lose their partition table association on upgrade, affecting rebuilds.
   Edit and re-save the host to fix it ([#11443](http://projects.theforeman.org/issues/11443)).
 
-If you are upgrading across more than one version, follow upgrade
-instructions for all previous versions for additional steps which may be
-necessary.
+When upgrading across more than one version it is recommended to upgrade to each 
+intermediate version and follow all upgrade instructions for the previous versions
+before updating to {{page.version}}
```
Comment (domcleal) [inline _includes/manuals/1.9/3.6_upgrade.md]: Missing a full stop/period on the end of all of these lines.

**Comment by domcleal** [comment]
:+1:

**Comment by vollmerk** [comment]
Fixed missing period at the end of the sentence.

**Comment by domcleal** [comment]
Thanks @vollmerk!

**Pull request merged**